src/pages/docs/projects/variables/system-variables.md

```diff
@@ -1077,7 +1078,7 @@ Example: *True*
 
 `Octopus.Action.Azure.DeploymentLabel`
 
-If set, the custom deployment label will be used for the Azure cloud service deployment. **Introduced in Calamari version 3.4.1.**.
```

Comment:
This version would be quite old now and this variable is practically in all modern versions of Octopus Server. I don't think it's worth tracking down and noting which version of Server this corresponds to.